### PR TITLE
DatePicker fix highlight in range

### PR DESCRIPTION
**Pull request by Fercas123**

### CODE REVIEW

**Review by origami-z — COMMENTED** [review]
Add a test?

File: packages/lab/src/__tests__/__e2e__/date-picker/DatePicker.cy.tsx [hunk 1]
```diff
@@ -257,7 +258,21 @@ describe("GIVEN a DatePicker", () => {
       cy.findByRole("button", { name: "Open Calendar" }).realClick();
       cy.findAllByRole("application").should("have.length", 1);
     });
-
+    it.only("should show hover all the first month when hovering through the second one ", () => {
```
Comment (origami-z) [inline packages/lab/src/__tests__/__e2e__/date-picker/DatePicker.cy.tsx:261]: `only`

Comment (Fercas123) [reply packages/lab/src/__tests__/__e2e__/date-picker/DatePicker.cy.tsx:261]: 🤦‍♀️ i do this every time

Comment (Fercas123) [inline packages/lab/src/__tests__/__e2e__/date-picker/DatePicker.cy.tsx:261]:
```suggestion
    it("should show hover all the first month when hovering through the second one ", () => {
```

**Review by lilyvc — APPROVED** [review]

**Pull request merged**